Packs/GroupIB_ThreatIntelligenceAttribution/Integrations/GroupIBTIA/test_data/example.json

```
@@ -292,62 +292,110 @@
-                            "authorEmail": "some@gmail.ru",
-                            "authorName": "sadsdsa",
-                            "dateCreated": "2019-06-20T13:13:35+00:00"
+                {
+                    "authorEmail": "some@email.com",
+                    "authorName": "somename"
+                }
+            ],
+            "dataFound": {
+                "password": 8,
+                "apikey": 2,
+                "secret": 1
+            },
+            "dateCreated": "2021-01-23T22:12:58+03:00",
+            "dateDetected": "2021-01-28T22:32:54+03:00",
+            "evaluation": {
+                "admiraltyCode": "A1",
+                "credibility": 50,
+                "reliability": 50,
+                "severity": "orange",
+                "tlp": "amber",
+                "ttl": 30
+            },
+            "favouriteForCompanies": [],
+            "files": [
+                {
+                    "dataFound": [],
+                    "dateCreated": "1970-01-01T03:00:00+03:00",
+                    "dateDetected": "2021-01-28T19:37:08+00:00",
+                    "evaluation": {
+                        "admiraltyCode": "A1",
+                        "credibility": 30,
+                        "reliability": 100,
+                        "severity": "gray",
+                        "tlp": "amber",
+                        "ttl": 30
+                    },
+                    "id": "9ea4c62e9e5da6bc86ecb35e9f4da12fd29cfa5e",
+                    "matchesType": [
+                        "readme"
+                    ],
+                    "matchesTypeCount": {
+                        "readme": 1
+                    },
+                    "name": "example.txt",
+                    "revisions": [
+                        {
+                            "bind": [
+                                {
+                                    "bindBy": "",
+                                    "companyId": 0,
+                                    "data": "",
+                                    "ruleId": 0,
+                                    "type": "readme"
+                                }
+                            ],
+                            "data": null,
+                            "hash": "21e02c97e2ecdabb272bfefbe29fd3f50032f525",
+                            "info": {
+                                "authorEmail": "some@email.com",
+                                "authorName": "TEST",
+                                "timestamp": 1611429178
+                            }
                         }
+                    ],
+                    "rules": null,
+                    "url": "https://github.com/somegit"
                     }
-                ],
-                "seqUpdate": 1611219371626093,
-                "source": "github"
-            }
+            ],
+            "hideForCompanies": [],
+            "id": "21aed9b86d2e6cbb15180d803a84f6d27f673db4",
```

Or add it to secrets-ignore
```suggestion
            "id": "1111111111111111111111111111111111111111",
```